src/pages/EntityDetails/Model/ActionLogs/ActionLogs.test.js

```diff
@@ -78,6 +116,6 @@ describe("Action Logs", () => {
         result[rowIndex].push(cell.text());
       });
     });
-    expect(expected).toEqual(result);
+    expect(result).toEqual(expected);
   });
```

Comment:
It'd probably be good to add a test for checking that changing the the output selector changes the message col.

Reply:
Good call!